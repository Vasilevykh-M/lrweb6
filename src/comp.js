export const difDiv = ({params}) => {return <div>{params.text}</div>;};
export const difH1 = ({params}) => {return <h1>{params.text}</h1>;};
export const difH2 = ({params}) => {return <h2>{params.text}</h2>;};