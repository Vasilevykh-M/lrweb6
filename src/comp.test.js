import renderer from 'react-test-renderer';
import {difDiv, difH1, difH2} from './comp.js'

it("1", ()=>{
    const result = renderer.create(<difDiv
    text = {"text"}
    />);
    expect(result).toMatchSnapshot();
});

it("2", ()=>{
    const result = renderer.create(<difH1
        text = {"text"}
    />);
    expect(result).toMatchSnapshot();
});

it("3", ()=>{
    const result = renderer.create(<difH2
        text = {"text"}
    />);
    expect(result).toMatchSnapshot();
});