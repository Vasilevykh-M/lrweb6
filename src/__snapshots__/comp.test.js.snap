// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 1`] = `
<difDiv
  text="text"
/>
`;

exports[`2 1`] = `
<difH1
  text="text"
/>
`;

exports[`3 1`] = `
<difH2
  text="text"
/>
`;
